doc/src/examples/algorithms/make_2d_point.cpp

```diff
@@ -39,10 +38,9 @@ void construct_and_display()
 
 int main()
 {
-    construct_and_display<boost::geometry::model::d2::point_xy<double> >();
-    construct_and_display<boost::geometry::model::d2::point_xy<int> >();
-    construct_and_display<boost::tuple<double, double> >();
-    construct_and_display<boost::polygon::point_data<int> >();
+    construct_and_display<boost::geometry::model::d2::point_xy<double>>();
+    construct_and_display<boost::geometry::model::d2::point_xy<int>>();
+    construct_and_display<boost::tuple<double, double>>();
```

Comment:
`Boost.Polygon` is removed. The dependency is removed. This is probably not a use case for anyone, anymore.